lxd/db/cluster/stmt.go

```diff
@@ -51,6 +51,16 @@ func Stmt(tx *sql.Tx, code int) *sql.Stmt {
 	return tx.Stmt(stmt)
 }
 
+// StmtString returns the in-memory query string with the given code.
+func StmtString(code int) string {
+	stmt, ok := stmts[code]
+	if !ok {
+		panic(fmt.Sprintf("No prepared statement registered with code %d", code))
```

Comment:
We don't panic in LXD as it will crash the whole daemon.

Reply:
The other functions that access the `stmts` map panicked. Shall I just ignore the case, or switch them all to errors?

Reply:
Please can you switch them all to errors as that must have been there for a while and I didn't know about it :)

Reply:
There's actually about 28 panics in LXD at the moment. I'll add an issue to get rid of them.